core/sched.c

```diff
@@ -162,15 +162,15 @@ void sched_set_status(thread_t *process, unsigned int status)
 {
     if (status >= STATUS_ON_RUNQUEUE) {
         if (!(process->status >= STATUS_ON_RUNQUEUE)) {
-            DEBUG("sched_set_status: adding thread %" PRIkernel_pid " to runqueue %" PRIu16 ".\n",
+            DEBUG("sched_set_status: adding thread %" PRIkernel_pid " to runqueue %" PRIu8 ".\n",
```

Comment:
I believe PRIu8 has some troubles as well with nano printf.

Reply:
A solution could be to just cast the argument to unsigned int instead

Reply:
mhm, but if so we need to change that in some more places through out RIOT, I found ~20 usages. If there is an actual issue with nano printf, we should fix all those in a separate PR *and* have a tests to check that in the future. 

So I would leave as is here for now, okay?